fluss-lake/fluss-lake-lance/src/test/java/com/alibaba/fluss/lake/lance/tiering/LanceArrowWriterTest.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.alibaba.fluss.lake.lance.tiering;
+
+import com.alibaba.fluss.lake.lance.utils.LanceArrowUtils;
+import com.alibaba.fluss.record.GenericRecord;
+import com.alibaba.fluss.record.LogRecord;
+import com.alibaba.fluss.row.GenericRow;
+import com.alibaba.fluss.types.DataField;
+import com.alibaba.fluss.types.DataTypes;
+import com.alibaba.fluss.types.RowType;
+
+import org.apache.arrow.memory.BufferAllocator;
+import org.apache.arrow.memory.RootAllocator;
+import org.apache.arrow.vector.VectorSchemaRoot;
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.concurrent.atomic.AtomicInteger;
+
+import static com.alibaba.fluss.record.ChangeType.APPEND_ONLY;
+import static org.assertj.core.api.Assertions.assertThat;
+
+/** The UT for Lance Arrow Writer. */
+public class LanceArrowWriterTest {
+    @Test
+    public void testLanceArrowWriter() throws Exception {
+        try (BufferAllocator allocator = new RootAllocator(Long.MAX_VALUE)) {
+            List<DataField> fields = Arrays.asList(new DataField("column1", DataTypes.INT()));
+
+            RowType rowType = new RowType(fields);
+            final int totalRows = 125;
+            final int batchSize = 34;
+
+            final LanceArrowWriter arrowWriter =
+                    new LanceArrowWriter(
+                            allocator, LanceArrowUtils.toArrowSchema(rowType), batchSize, rowType);
+            AtomicInteger rowsWritten = new AtomicInteger(0);
+            AtomicInteger rowsRead = new AtomicInteger(0);
+
+            Thread writerThread =
```

Comment:
The test logic looks strange to me. Do we really need this test? I prefer to delete this test since we have LanceTieringTest